client/config/environment.js

```diff
@@ -4,12 +4,14 @@ var devHost = 'http://localhost:1337';
 var prodHost = 'http://gallery.hairbrain.io';
 var devS3 = 'https://gallery13-dev.s3.amazonaws.com';
 var prodS3 = 'https://gallery13.s3.amazonaws.com';
+var gaId = 'UA-12392326-1';
 
 module.exports = function(environment) {
   var ENV = {
     modulePrefix: 'client',
     podModulePrefix: 'client/pods',
     environment: environment,
+    isProduction: (this.environment === 'production'),
```

Comment:
I can make those changes when I get back home and will attempt to squash my commits :) then we can merge this one in.